pkg/collector/corechecks/oracle/config/README.TXT

```diff
@@ -0,0 +1,22 @@
+Currently supported configurations
```

Comment:
This looks like markdown, could we rename the file to `README.md`. Also, all the other README of the repo use markdown.

Reply:
I renamed README.TXT to README.md.